CoC.md

```diff
@@ -10,3 +12,16 @@
 8. Крайне нежелательно дублировать один вопрос в нескольких чатах.
 9. Админы чата могут удалить любое сообщение, не соответствующее правилам.
 10. В рабочих чатах лучше придерживаться бизнес-стиля общения т.е не использовать обсцессивную лексику.
+
+## Code of conduct
+
+1. It is not recommended to write many small messages in a row connected with one thought. State your thoughts in one big message. For large messages, it’s easier to track a thread. Otherwise, it can be considered by the administrator as a flood.
+2. Reply to messages. All official customers support this feature. When there are several threads in a chat at the same time, it is impossible to understand what message is addressed to whom.
+3. Insulting chat participants and admins is undesirable. Offensive messages will be deleted, and in case of a malicious violation will be banned (but this is inaccurate).
```

Comment:
and in case of a malicious violation, a user will be banned (but this is inaccurate).